lib/cli/interactive-setup/console-dev-mode.js

```diff
@@ -0,0 +1,337 @@
+  return new Date(endTime).getTime() - new Date(startTime).getTime();
+};
+
+const capitalizeFirstLetter = (string) => string.charAt(0).toUpperCase() + string.slice(1);
+
+const formatDuration = (milliseconds) => {
+  const seconds = milliseconds / 1000;
+  const minutes = seconds / 60;
+
+  if (milliseconds <= 999) {
+    return `${Math.round(milliseconds)}ms`;
+  } else if (seconds <= 59) {
+    return `${Math.floor(seconds * 100) / 100}s`;
+  } else if (minutes >= 1) {
+    return `${Math.floor(minutes * 100) / 100}min`;
+  }
+  return 'n/a';
+};
+
+const formatAWSSDKName = ({ activity }) => {
+  const { name } = activity;
+  const [, , service, operation] = name.split('.');
+  let finalName = `AWS SDK • ${capitalizeFirstLetter(service)} • ${operation.toUpperCase()}`;
+  if (name.includes('aws.sdk.dynamodb')) {
+    finalName = `AWS SDK • DynamoDB • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.eventbridge')) {
+    finalName = `AWS SDK • Event Bridge • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.secretsmanager')) {
+    finalName = `AWS SDK • Secrets Manager • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.kinesis')) {
+    finalName = `AWS SDK • Kinesis • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.elastictranscoder')) {
+    finalName = `AWS SDK • Elastic Transcoder • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.iotdata')) {
+    finalName = `AWS SDK • IOT Data • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.kinesisvideo')) {
+    finalName = `AWS SDK • Kinesis Video • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.sns')) {
+    finalName = `AWS SDK • SNS • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.sqs')) {
+    finalName = `AWS SDK • SQS • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.ssm')) {
+    finalName = `AWS SDK • SSM • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.s3')) {
+    finalName = `AWS SDK • S3 • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.elb')) {
+    finalName = `AWS SDK • ELB • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.emr')) {
+    finalName = `AWS SDK • EMR • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.es')) {
+    finalName = `AWS SDK • ES • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.fms')) {
+    finalName = `AWS SDK • FMS • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.ecs')) {
+    finalName = `AWS SDK • ECS • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.ec2')) {
+    finalName = `AWS SDK • EC2 • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.eks')) {
+    finalName = `AWS SDK • EKS • ${operation.toUpperCase()}`;
+  } else if (name.includes('aws.sdk.ebs')) {
+    finalName = `AWS SDK • EBS • ${operation.toUpperCase()}`;
+  }
```

Comment:
Is this a fixed list? What if we add new service, will we remember to update this module?

Maybe let's come up with one generic method and iterate just on those which require specific translation

Reply:
I think the trouble here is that the names are kind of curated. 

e.g dynamodb = DynamoDB, eventbridge = Event Bridge, s3 = S3. 

There is a fallback if a service doesn’t exist in this curated list so that it will just use the uppercase version of the service in the `aws.sdk.service`.

I could probably just remove some of the unnecessary formats here but I am not sure this is a way to not do some sort of curation here. 🤔

Reply:
Ok, yes, so in that case, let's just remove those cases which are well addressed by the fallback